kolla-ansible/ubuntu/docker/trilio-datamover/nova-sudoers

```diff
@@ -1,2 +1 @@
-nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```

Comment:
@amitkumar-a4 @abhijeetpatra : Let me know, if this change is fine. This is optional. But to remove unnecessary privileges, I am removing this from sudoers.